sandbox/templates/djangocms_video/full-width/video_player.html

```diff
@@ -0,0 +1,3 @@
+<div class="embed-responsive embed-responsive-16by9">
```

Comment:
same question as previous PR: can we put these templates and scss files in Richie? That's because we want Richie to work like this by default without having to do these overrides in your project...

Reply:
@sampaccoud for third party plugins we can't, since they use a strange template path system which does not allow to define a real path just a keyword to find it.

Reply:
It was agreed to move all templates/scss/static files out of the sandbox and to the core richie app in a separate PR so that it is part of a startup kit that installs Richie with nearly zero work.

Reply:
FYI @jmaupetit

Reply:
:ok_hand: